lib/kennel/utils.rb

```diff
@@ -5,6 +5,7 @@ module Utils
 
     class TeeIO < IO
       def initialize(ios)
+        super(0) # called with fake file descriptor 0, so we can call super and get a proper class
```

Comment:
FD 0 isn't "fake", it's stdin. Does it _matter_ that it's stdin? Would 1 (stdout) work just as well? If not, why not? Does this even need to be a subclass of `IO`?

The code _might_ be fine; the comment leaves a lot of questions unanswered.

Reply:
I'm also of course aware that `TeeIO` isn't a new class, but the point stands regardless.